src/containers/HeaderContainer/HeaderContainer.vue

```diff
@@ -249,11 +268,14 @@ export default {
       gasPrice: '0',
       error: {},
       resolver: () => {},
-      showGettingStarted: ''
+      showGettingStarted: '',
+      address: '',
+      blockExplorerURL: ''
     };
   },
   computed: {
     ...mapGetters({
+      network: 'network',
       wallet: 'wallet',
```

Comment:
lets take out all the unused getters, wallet, online..etc

Reply:
Remove 'online' from mapGetters.